ops/docker/ci-builder/Dockerfile

```diff
@@ -84,10 +84,12 @@ RUN echo "downloading solidity compilers" && \
   rm solc-linux-amd64-v0.8.12+commit.f00d7308
 
 RUN echo "downloading and verifying Codecov uploader" && \
-  curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step && \
+  curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import && \
   curl -Os "https://uploader.codecov.io/latest/linux/codecov" && \
   curl -Os "https://uploader.codecov.io/latest/linux/codecov.SHA256SUM" && \
   curl -Os "https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig" && \
   gpgv codecov.SHA256SUM.sig codecov.SHA256SUM && \
   shasum -a 256 -c codecov.SHA256SUM || sha256sum -c codecov.SHA256SUM && \
-  chmod +x codecov
+  cp codecov /usr/local/bin/codecov && \
+  chmod +x /usr/local/bin/codecov  && \
+  rm codecov
```

Comment:
I did my best here to follow the convention used above, ie.
1. `check-changed` is placed in `/usr/local/bin` so I did that here too.
2. I used the same `cp`/`rm` steps used for the `solc` bins above.

Reply:
Great, thank you!